tests/test_athena_cache.py

```diff
@@ -1,17 +1,30 @@
 import logging
+from importlib import reload
+from types import ModuleType
+from typing import Iterator
 from unittest.mock import patch
 
 import pandas as pd
 import pytest
 
-import awswrangler as wr
-
 from ._utils import ensure_athena_query_metadata
 
 logging.getLogger("awswrangler").setLevel(logging.DEBUG)
 
 
-def test_athena_cache(path, glue_database, glue_table, workgroup1):
+@pytest.fixture(scope="function")
```

Comment:
Does it make sense to push this to our conftest or would it not work if it's there?

Reply:
I can try it. I was worried that it would lead to scoping issues, as `wr` would be defined both as an import and as a fixture. But I suppose a fixture isn't invoked until the unit test is started.